Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPISidebarCocoa.mm

```diff
@@ -32,34 +32,37 @@
 
 #if ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
 
+#import "WKWebExtensionControllerDelegatePrivate.h"
 #import "WebExtensionSidebar.h"
 
 namespace WebKit {
 
+static NSString * const unknownErrorString = @"an unknown error occurred";
+
 template<typename T>
 static Expected<T, WebExtensionError> toExpected(std::optional<T>&& optional, NSString * const errorMessage = @"value not found")
 {
     if (optional)
         return WTFMove(optional.value());
-    return toWebExtensionError(nil, nil, errorMessage);
+    return makeUnexpected(errorMessage);
 }
 
 static Expected<Ref<WebExtensionSidebar>, WebExtensionError> getSidebarWithIdentifiers(std::optional<WebExtensionWindowIdentifier> windowIdentifier, std::optional<WebExtensionTabIdentifier> tabIdentifier, WebExtensionContext& context)
 {
     if (windowIdentifier && tabIdentifier)
-        return toWebExtensionError(nil, nil, @"cannot specify both windowId and tabId");
+        return makeUnexpected(@"it is illegal to specify both windowId and tabId");
```

Comment:
```suggestion
        return makeUnexpected(@"it cannot specify both 'windowId' and 'tabId'");
```

Reply:
My wording really requires a key to be passed to `toWebExtensionError` in the second param. An example from scripting:

```objc++
toErrorString(nil, @"details", @"it cannot specify both 'func' and 'function'. Please use 'func'")
```